app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Tasky</string>
+    <string name="login_greetings">Welcome Back!</string>
+    <string name="login_main_button">LOG IN</string>
+    <string name="login_sign_up">DON’T HAVE AN ACCOUNT? SIGN UP</string>
```

Comment:
Personally, I don't separate the strings based on the screen

I would go with something like:
```
<string name="welcome_back">Welcome Back!</string>
<string name="log_in">LOG IN</string>
<string name="dont_have_an_account">DON’T HAVE AN ACCOUNT? SIGN UP</string>
```

Reply:
I agree, the disadvantage of separating them based on screens is that as soon as a string needs to be used for multiple screens, it becomes messy or you have duplicated resources which ends up with the same